web/src/components/countrytable.js

```diff
@@ -412,15 +409,14 @@ const CountryElectricityProductionTable = React.memo(({
             index={index}
             label={d.mode}
             width={width}
+            electricityMixMode={electricityMixMode}
             isMobile={isMobile}
             data={data}
           >
             <image
-              width={FLAG_SIZE}
-              height={FLAG_SIZE}
               style={{ pointerEvents: 'none' }}
               x={exchangeFlagX}
-              xlinkHref={flagUri(d.mode, FLAG_SIZE)}
+              xlinkHref={flagUri(d.mode)}
```

Comment:
nice

Reply:
By the way, I noticed that a lot of the current Sentry errors are somehow caused by `flagUri` not checking for `undefined` - some of the changes I made in this PR will hopefully resolve some of these issues.